_posts/2020/2020-05-15-constraints-and-freedom-writing-perfect-program.md

```diff
@@ -83,9 +83,8 @@ public class Sum {
 }
 ```
 
-## Well... Not really
-We run our program and start manually testing it (we don't want to write unit tests to keep our program as small as possible, but keep in mind that this is a must if we want to make our program perfect in every way).
-This gives us the following:
+We ran our program and started manually testing it (we didn't want to write unit tests to keep our program as small as possible, but keep in mind that this is a must if we want to make our program perfect in every way).
+This gave us the following:
```

Comment:
Same here, trying to find the solution together with the reader to keep him entertained